x-pack/legacy/plugins/siem/public/components/navigation/tab_navigation/index.test.tsx

```diff
@@ -74,8 +74,8 @@ describe('Tab Navigation', () => {
       expect(hostsTab.prop('isSelected')).toBeTruthy();
     });
     test('it changes active tab when nav changes by props', () => {
-      const wrapper = shallow(<TabNavigation {...mockProps} />);
-      const networkTab = () => wrapper.find('[data-test-subj="navigation-network"]');
+      const wrapper = mount(<TabNavigation {...mockProps} />);
+      const networkTab = () => wrapper.find('[data-test-subj="navigation-network"]').first();
```

Comment:
I do not understand why this test will change, I am confused

Reply:
Turns out that there is a bug in React related to updating state in shallow wrappers, if we just replace shallow with mount, we’re good to go.

Reply:
https://itnext.io/testing-components-built-using-react-hooks-with-jest-enzyme-edb87d703756